src/main/java/nextstep/courses/domain/SessionState.java

```diff
@@ -12,13 +15,23 @@ public enum SessionState {
     }
 
     public static SessionState convert(String code) {
+
+        if(OldVersion.stream()
+                .anyMatch(s -> s.getCode().equals(code))){
+            return PREPARING;
+        }
+
         return Arrays.stream(SessionState.values())
                 .filter(s -> s.code.equals(code))
                 .findFirst()
                 .orElse(CLOSE);
     }
 
-    public boolean isRecruitable() {
-        return this.equals(RECRUITING);
+    public static boolean isProceeding(SessionState sessionState) {
+        return sessionState.equals(PROCEEDING);
+    }
```

Comment:
```suggestion
    public boolean isProceeding() {
        return  this == PROCEEDING;
    }
```

enum도 jvm 상에 유일하게 하나의 인스턴스를 보장하는 클래스다.
이 기준으로 구현하면 클래스와 관련한 모든 OOP 원칙을 따라 구현하면 된다.